daphne/src/lib.rs

```diff
@@ -227,13 +238,19 @@ pub struct ProblemDetails {
 }
 
 /// Per-task DAP parameters.
+//
+// TODO spec: `max_window_size`, `min_duration_start`, and `max_duration_end`
+//            should be reflected to the DAP spec.
 #[derive(Deserialize, Serialize)]
 #[serde(try_from = "ShadowDapTaskConfig")]
 pub struct DapTaskConfig {
     pub leader_url: Url,
     pub helper_url: Url,
     pub min_batch_duration: u64, // seconds
     pub min_batch_size: u64,     // number of reports
+    pub max_window_size: u64,    // number of acceptable batches of reports
```

Comment:
Let's do this instead
```suggestion
    /// Number of acceptable batches of reports.
    pub max_window_size: u64,
```

Also, this comment is not very descriptive. It's especially important to be clear here since this isn't in the spec.

Reply:
Added a more descriptive comment and moved the three variables out of task_config and into their own config struct.